dataset/dictionary/items.json5

```diff
@@ -1198,6 +1198,34 @@
     notes: "「神の碁盤」に配置するアイテム",
   },
 
+  {
+    en: "Plume of Purifying Light",
+    ja: "浄光の羽根",
+    zhCN: "净光翎",
```

Comment:
@Bill-Haku @SleepyAsh0191 Can you review if these Chinese translations are correct when you have time?

Reply:
This is correct.

Reply:
@Bill-Haku Thanks for the confirmation!